examples/tracing/client.py

```diff
@@ -0,0 +1,97 @@
+"""
+This example demonstrates how to create a trace with multiple spans using the low-level MLflow client APIs.
+"""
+import json
+
+import mlflow
+
+exp = mlflow.set_experiment("mlflow-tracing-example")
+exp_id = exp.experiment_id
+
+# Initialize MLflow client to interact with the tracking server.
```

Comment:
```suggestion
# Initialize MLflow client.
```

I'd remove `interact with the tracking server` because we're not running the tracking server in this example.